src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NFloat.cs

```diff
@@ -701,5 +710,597 @@ public int CompareTo(object? obj)
+        }
+
+        /// <inheritdoc cref="IBitwiseOperators{TSelf, TOther, TResult}.op_ExclusiveOr(TSelf, TOther)" />
+        static NFloat IBitwiseOperators<NFloat, NFloat, NFloat>.operator ^(NFloat left, NFloat right)
+        {
+#if TARGET_32BIT
+            uint bits = BitConverter.SingleToUInt32Bits(left._value) ^ BitConverter.SingleToUInt32Bits(right._value);
+            NativeType result = BitConverter.UInt32BitsToSingle(bits);
+            return new NFloat(result);
+#else
+            ulong bits = BitConverter.DoubleToUInt64Bits(left._value) ^ BitConverter.DoubleToUInt64Bits(right._value);
+            NativeType result = BitConverter.UInt64BitsToDouble(bits);
+            return new NFloat(result);
+#endif
+        }
+
+        /// <inheritdoc cref="IBitwiseOperators{TSelf, TOther, TResult}.op_OnesComplement(TSelf)" />
+        static NFloat IBitwiseOperators<NFloat, NFloat, NFloat>.operator ~(NFloat value)
+        {
+#if TARGET_32BIT
+            uint bits = ~BitConverter.SingleToUInt32Bits(value._value);
+            NativeType result = BitConverter.UInt32BitsToSingle(bits);
+            return new NFloat(result);
+#else
+            ulong bits = ~BitConverter.DoubleToUInt64Bits(value._value);
+            NativeType result = BitConverter.UInt64BitsToDouble(bits);
+            return new NFloat(result);
+#endif
+        }
+
+        //
+        // IDecrementOperators
+        //
+
+        /// <inheritdoc cref="IDecrementOperators{TSelf}.op_Decrement(TSelf)" />
+        static NFloat IDecrementOperators<NFloat>.operator checked --(NFloat value) => --value;
+
+        //
+        // IDivisionOperators
+        //
+
+        /// <inheritdoc cref="IDivisionOperators{TSelf, TOther, TResult}.op_CheckedDivision(TSelf, TOther)" />
+        static NFloat IDivisionOperators<NFloat, NFloat, NFloat>.operator checked /(NFloat left, NFloat right) => left / right;
+
+        //
+        // IExponentialFunctions
+        //
+
+        /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp" />
+        public static NFloat Exp(NFloat x) => new NFloat(NativeType.Exp(x._value));
+
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.ExpM1(TSelf)" />
+        // public static NFloat ExpM1(NFloat x) => new NFloat(NativeType.ExpM1(x._value));
+
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp2(TSelf)" />
+        // public static NFloat Exp2(NFloat x) => new NFloat(NativeType.Exp2(x._value));
+
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp2M1(TSelf)" />
+        // public static NFloat Exp2M1(NFloat x) => new NFloat(NativeType.Exp2M1(x._value));
+
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp10(TSelf)" />
+        // public static NFloat Exp10(NFloat x) => new NFloat(NativeType.Exp10(x._value))(x);
+
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp10M1(TSelf)" />
+        // public static NFloat Exp10M1(NFloat x) => new NFloat(NativeType.Exp10M1(x._value));
```

Comment:
What are these and why are they commented out?

Reply:
These are functions that are approved for `IExponentialFunctions` but which aren't implemented yet.

They'll be coming sometime later this week but will miss Preview 4.